src/tools.ts

```diff
@@ -1,6 +1,13 @@
 import * as monaco from 'monaco-editor'
 
-function isPasteAction (handlerId: string, payload: unknown): payload is monaco.editor.PastePayload {
+interface PastePayload {
+  text: string
+  pasteOnNewLine: boolean
+  multicursorText: string[] | null
+  mode: string | null
+}
```

Comment:
Why don't we use monaco type anymore ?

Reply:
This type is defined internally in vscode, it's not even exported

Our custom build of monaco-editor was exporting it but we don't have it now